src/search-manager/SearchFilterWidget.tsx

```diff
@@ -27,60 +25,47 @@ const SearchFilterWidget: React.FC<{
   children: React.ReactNode;
   clearFilter: () => void,
   icon: React.ComponentType;
-}> = ({ appliedFilters, ...props }) => {
+}> = ({
+  appliedFilters, children, clearFilter, ...props
+}) => {
   const intl = useIntl();
-  const [isOpen, open, close] = useToggle(false);
-  const [target, setTarget] = React.useState<HTMLButtonElement | null>(null);
 
   const clearAndClose = React.useCallback(() => {
-    props.clearFilter();
-    close();
-  }, [props.clearFilter]);
+    clearFilter();
+  }, [clearFilter]);
```

Comment:
nit: since `clearAndClose` is now just calling `clearFilter`, you can replace its usage below with `clearFilter`, with no need to create this extra callback.